test/orm/declarative/test_typing_py3k.py

```diff
@@ -0,0 +1,23 @@
+from typing import Generic
+from typing import Type
+from typing import TypeVar
+
+from sqlalchemy import Column
+from sqlalchemy import Integer
+from sqlalchemy.orm import as_declarative
+
+
+def test_class_getitem():
```

Comment:
this is good, but for it to run in our suite it has to use our base, so change this as:

from sqlalchemy.testing import fixtures

class DeclarativeBaseTest(fixtures.TestBase):
    def test_class_getiem(self):
       # ...


then we are good.   great job!